### PR TITLE
Html::add is deprecated

### PR DESCRIPTION
**Pull request by CZechBoY**
in Nette 2.4 method Html::add is deprecated => replaced with Html::addHtml

### CODE REVIEW

File: cs/html-elements.texy [hunk 1]
```diff
@@ -39,7 +39,7 @@ $el = Html::el('input', [
 	'class' => ['red', 'important'],
 ]);
 
-$el->addAttributes([ // hromadné nastavení dalších atributů
+$el->Attributes([ // hromadné nastavení dalších atributů
```
Comment (matej21) [inline cs/html-elements.texy]: tohle si asi upravit nechtel :)

Comment (CZechBoY) [inline cs/html-elements.texy]: hm, ne :D
díky, opravim

Comment (CZechBoY) [inline cs/html-elements.texy]: @matej21 opraveno :)

**Comment by dg** [comment]
Thanks

File: cs/html-elements.texy (hunk 1 above)
Comment (CZechBoY) [inline cs/html-elements.texy]: yes, it was misreplaced as matej21 pointed in first review.

**Pull request merged**